tsdb/functions.go

```diff
@@ -1720,11 +1720,24 @@ func (a rawOutputs) Swap(i, j int)      { a[i], a[j] = a[j], a[i] }
 func greaterThan(a, b interface{}) bool {
 	switch t := a.(type) {
 	case int64:
-		return t > b.(int64)
+		switch bb := b.(type) {
```

Comment:
Really minor style note: I'm not sure if we're following this convention elsewhere in the code base, but [Effective Go](https://golang.org/doc/effective_go.html#type_switch) says it's "idiomatic to reuse the name in such cases, in effect declaring a new variable with the same name but a different type in each case." i.e. `switch b := b.(type)` rather than `bb`.